src/main/java/io/jenkins/plugins/synopsys/security/scan/global/ApplicationConstants.java

```diff
@@ -64,6 +67,14 @@ public class ApplicationConstants {
     public static final String NETWORK_AIRGAP_KEY = "network_airgap";
     public static final String RETURN_STATUS_KEY = "return_status";
 
+    public static final String BLACKDUCK_REPORTS_SARIF_CREATE_KEY = "blackduck_reports_sarif_create";
+
+    public static final String BLACKDUCK_REPORTS_SARIF_FILE_PATH_KEY = "blackduck_reports_sarif_file_path";
+
+    public static final String BLACKDUCK_REPORTS_SARIF_GROUPSCAISSUES_KEY = "blackduck_reports_sarif_groupSCAIssues";
+
+    public static final String BLACKDUCK_REPORTS_SARIF_SEVERITIES_KEY = "blackduck_reports_sarif_severities";
+
```

Comment:
Move this segment up, so that all the Blackduck related parameters are together.